packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -91,6 +99,7 @@ Limitations of JSON which Qwik solves:
   - `Map` and `Set` instances
 
 Limitations of JSON that Qwik does not solve:
+[//]: # (What does it mean ? Some parts of an app would not be serialized / resumed ? Is there a plan B ?)
```

Comment:
Well, if it can't be serialized, it should be only run on the client first, ,so my guess is `useClientEffect$` should be used in those occasions